config/environment.js

```diff
@@ -79,5 +79,9 @@ module.exports = function(environment) {
   // Whether to validate survey forms
   ENV.validate = false;
 
+  // Whether to take the participant to the last page they were on
+  // when they exited the study. If false, start from the beginning.
+  ENV.continueSession = false;
```

Comment:
Should this also be moved into the feature flags section that is being created as part of https://github.com/CenterForOpenScience/exp-addons/pull/168 / #51 ? If I merge those first, then this PR can be updated with minimal merge conflicts.